src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
+    @Test
+    public void allFilesExist() {
+        assertTrue(CertificateUtils.verifyCertificatesExist("src/test/resources/testVerifyCertificatesExist/allFiles"));
```

Comment:
good practice to place them under the method name
`src/test/resources/com/github/dockerjava/core/util/CertificateUtilsTest/allFilesExist `